helm/net-exporter/values.yaml

```diff
@@ -12,6 +12,14 @@ dns:
   namespace: kube-system
   service: coredns
 
+dnscache:
+  port: 53
+
+cluster:
+  kubernetes:
+    DNS:
+      IP: 172.31.0.10
```

Comment:
Is this always the case?

Reply:
nope, that's the default.
We do the same in the coredns app here https://github.com/giantswarm/coredns-app/blob/master/helm/coredns-app/values.yaml#L62

if the value is different, the value gets injected in `values.yaml` by out app installation pipeline